hathor/conf/get_settings.py

```diff
@@ -33,6 +33,10 @@ class _SettingsMetadata(NamedTuple):
 _settings_singleton: Optional[_SettingsMetadata] = None
 
 
+def get_settings() -> Settings:
```

Comment:
Should we cache and always return the same instance?

Reply:
We do, the current `HathorSettings()` function caches the same instance in the `_settings_singleton` attribute that you can see above. After the renaming process is complete, `HathorSettings()` will be removed and `get_settings()` will absorb its implementation.

Reply:
Got it. Thanks!